### ISSUE TITLE
3.4.0 Beta Testing Guide

### ISSUE THREAD

**Issue by ckmahoney** [warfare-plugins/social-warfare#668]
# Testing Guidance

**Zips**

`Core 3.3.99 - Last updated Monday, November 26 6:18 EST`
[social-warfare.zip](https://github.com/warfare-plugins/social-warfare/files/2617645/social-warfare.zip)

`Pro 3.3.97  - Last updated Monday, November 5:20 EST`
[social-warfare-pro.zip](https://github.com/warfare-plugins/social-warfare/files/2617529/social-warfare-pro.zip)

#### Timeline 
Internal Beta Testing: October 29 to November 2
Public Beta Testing: November 3 to November 9

#### Why test?
A plugin update is a release that provides fixes to old problems and supplies new features. In an ideal world these changes are perfect and work for everybody every time. 

In reality, the world has many different kinds of hosting environments and WordPress install setups. While the new code works like a charm on our development sites, we need to know that it works equally as well for other sites. 

#### How to test

All of our issues are logged in the milestone for the update. We would like you as a tester to click an issue, read the notes on how to test, 

Once tested and you have confirmed that everything is good, just give the opening comment on the issue a thumbs up emoji. There should already be at least one thumbs up on every issue from a Warfare Plugins member, so you can just click the existing one to add your 👍 

If something is wrong, then please describe it on that issue as a comment on the thread. 

# tl;dr

1. View the issues on our current milestone here: https://github.com/warfare-plugins/social-warfare/milestone/6
2. Select an issue you haven't tested yet that is marked with a green "READY FOR TESTING" tag.
3. Follow the instructions for testing.
4. Thumbs up the issue if it went well. If not, leave a note.

#### Protips

1. Install the plugin Query Monitor on your testing site. This plugin will reveal any errors or warnings as they occur while you browse your site. 

2. Ensure that your testing site has error logging enabled and that you know how to access the error log for your server. You can confirm this by checking your wp_config.php file. 

3. Begin browsing your site with the plugin in various configurations. Toggle some of the options ON/OFF, change some settings in the dropdown menus, **save your changes**, and browse some more and confirm that things look okay.

4. In the error log, search/find the term `social-warfare`. If it shows up in the logs, please let us know and copy the error message either as a comment on this issue, or on the relevant issue if you can find it. 
   - To access your error log, you can either visit /wp-content/error.log on your site (such as `https://example.com/wp-content/error.log`), or fetch it from your server via FTP.

**Comment by DustinWStout** [comment]
here are all the things I found yesterday, guys. I forgot to comment yesterday. If an item is checked off, it's because I added it to an appropriate issue already: 

- [ ] high share counts on side-floating button style overflows

- [x] need to add `padding: 30px 0;` to `.sw-header-menu li a` for the settings page
- [ ] Display > Share Counts > Alignment: Left doesn't work
- [ ] Registered plugin requires page reload to display Pro options in settings page
- [x] let's change the Display > Social Networks button hover text box to `#ced3dc`
- [ ] high share counts total on Boxes & Circles style overflows/gets cut off

- [ ] side-floating buttons don't disappear when static buttons are in view
- [ ] float before content setting doesn't work
- [ ] top/bottom-floating buttons don't display
- [x] pin image for browser extensions not working at all as far as I can tell
- [x] Change recommended Pinterest image size to 600x900 (or 1000x1500)
- [x] `p.description` margin to `15px 0;`
- [x] give `.wp-core-ui #social_warfare a.swpmb-add-media.button` a margin of `15px 0;`
